packages/react-native-aztec/android/build.gradle

```diff
@@ -12,6 +12,7 @@ buildscript {
         wordpressUtilsVersion = '1.22'
         espressoVersion = '3.0.1'
         aztecVersion = 'v1.3.45'
+        willPublishReactNativeAztecBinary = properties["publishReactNativeAztecVersion"]?.trim() as boolean
```

Comment:
Is this checking whether the property is there and returning `true` if so? Neat one-liner.

Reply:
Yeap, that's what it's doing.